runs/2021/pyladies-ostrava-podzim/info.yml

```
@@ -69,30 +69,30 @@ plan:
 - slug: expressions
   title: 'Výrazy a podmínky'
   date: 2021-09-22
-#   materials:
-#   - url: https://youtu.be/r0esiaghzrY?t=483
-#     title: (1:49) Neuzavřená závorka – video (+ domácí úkoly v první části videa)
-#   - lesson: adm/close-paren
-#
-#   - url: https://youtu.be/9aJcvOfleFs
-#     title: (7:50) První program – video
-#   - lesson: beginners/hello-world
-#
-#   - url: https://youtu.be/14-LVG9Edng
-#     title: (12:38) Print a chybové hlášky – video
-#   - lesson: beginners/print
-#
-#   - url: https://youtu.be/-zKws24FmCg
-#     title: (15:08) Proměnné – video
-#   - lesson: beginners/variables
-#
-#   - url: https://youtu.be/nNXDXuN6Smg
-#     title: (8:32) Vyhodnocování výrazů – video
-#   - lesson: beginners/expressions
-#
-#   - url: https://youtu.be/Q1YJqWzfnck
-#     title: (17:27) Porovnání – video
-#   - lesson: beginners/comparisons
+  materials:
+  - url: https://youtu.be/r0esiaghzrY
+    title: (9:52) Neuzavřená závorka – video (+ domácí úkoly v první části videa)
+  - lesson: adm/close-paren
+
+  - url: https://youtu.be/9aJcvOfleFs
+    title: (7:50) První program – video
+  - lesson: beginners/hello-world
+
+  - url: https://youtu.be/14-LVG9Edng
+    title: (12:38) Print a chybové hlášky – video
+  - lesson: beginners/print
+
+  - url: https://youtu.be/-zKws24FmCg
+    title: (15:08) Proměnné – video
+  - lesson: beginners/variables
+
+  - url: https://youtu.be/nNXDXuN6Smg
+    title: (8:32) Vyhodnocování výrazů – video
+  - lesson: beginners/expressions
+
+  - url: https://youtu.be/Q1YJqWzfnck
+    title: (17:27) Porovnání – video
+  - lesson: beginners/comparisons
```

Last year we also linked to this  cheatsheet, so it might be a good idea to include it again.
```suggestion
  - lesson: beginners/comparisons
  
  - title: Tahák na debuggování (PDF)
      url: https://pyvec.github.io/cheatsheets/errors/errors-cs.pdf
      type: cheatsheet
```